### PR TITLE
[16.0][Feature] add website_geoengine_store_locator

### PR DESCRIPTION
**Pull request by wouitmil**
This module extends the ``website_geoengine`` odoo module, to add a map snippet view for the website.
By default the map features are filtered to only display contact of type `store`.

### CODE REVIEW

**Comment by yvaucher** [comment]
Please fix the pre-commit and squash some of those commits (Most likely in one as the feature is adding a new module)

**Comment by wouitmil** [comment]
> Please fix the pre-commit and squash some of those commits (Most likely in one as the feature is adding a new module)

**Comment by wouitmil** [comment]
Hello @yvaucher pre-commit fixed and commits squashed

File: website_geoengine/models/res_partner.py [hunk 1]
```diff
@@ -0,0 +1,117 @@
+from odoo import _, api, fields, models
```
Comment (yvaucher) [inline website_geoengine/models/res_partner.py:4]: Missing a license header

File: website_geoengine/readme/CONTIBUTORS.rst [hunk 2]
```diff
@@ -0,0 +1,2 @@
+* Stéphane Brunner <stephane.brunner@camptocamp.com>
```
Comment (yvaucher) [inline website_geoengine/readme/CONTIBUTORS.rst]: Typo in this file name, that's why you don't appear in the readme file as contributor.

File: website_geoengine_store_locator/readme/CONTIBUTORS.rst [hunk 3]
```diff
@@ -0,0 +1,2 @@
+* Stéphane Brunner <stephane.brunner@camptocamp.com>
```
Comment (yvaucher) [inline website_geoengine_store_locator/readme/CONTIBUTORS.rst]: Some typo on the file name. (missing an R)

**Review by yvaucher — CHANGES_REQUESTED** [review]
Code LGTM

Only a couple of issue on files.
Some python files misses a copyright header + readme are missing the contributors because of a typo in some of the file names

File: website_geoengine_store_locator/readme/CONTIBUTORS.rst (hunk 3 above)
Comment (wouitmil) [reply website_geoengine_store_locator/readme/CONTIBUTORS.rst]: Corrected

File: website_geoengine/readme/CONTIBUTORS.rst (hunk 2 above)
Comment (wouitmil) [reply website_geoengine/readme/CONTIBUTORS.rst]: Corrected !

File: website_geoengine/models/res_partner.py (hunk 1 above)
Comment (wouitmil) [reply website_geoengine/models/res_partner.py:4]: I then also added license informations in the JS files

**Review by yvaucher — APPROVED** [review]

**Pull request merged**